### ISSUE TITLE
New Feature: 3 Card Spread

### ISSUE THREAD

**Issue by LiliCecilia23** [joshuavickstrom/TarotWebapp#26]
Create all mechanics for a user to get a 3 card tarot spread on the Tarot Pull page. Since this will be our only option for now we can just have it automatically go to this spread rather than having the user go through a choice of spreads. We can add in that step later when we need it. For this we will need:

- [x] Card Component
     - Deck Property (only RWS right now but we should still have it)
     - Back Property
- [ ] Shuffle & Cut buttons (these won't do anything real for now, eventually they will have animation (Bonus?)
- [x] Code to randomize which cards are selected
- [x] UI to display the reading (Each card and its associated meaning from the arcana.json and what the position in the spread means - once we hook up the AI we can revisit how this is done so we can give more in depth readings) 
- [x] UI to do a new pull
